devicetypes/smartthings/smartsense-multi.src/smartsense-multi.groovy

```diff
@@ -120,8 +123,18 @@ private Map parseSingleMessage(description) {
 		handlerName: handlerName,
 		isStateChange: isStateChange,
 		displayed: displayed(description, isStateChange)
-	]
-	log.debug "Parse results for $device: $results"
+	)
+
+	results << createEvent(
+		name: "status",
+		value: value,
+		unit: null,
+		linkText: linkText,
+		descriptionText: descriptionText,
+		handlerName: handlerName,
+		isStateChange: isStateChange,
```

Comment:
it is always true for phase 1, where status is in sync with the contact attribute. for phase 2, they will be independent events in garage door mode.